### PR TITLE
Update StringDictionary.vi.xaml

### PR DESCRIPTION
**Pull request by duyplus**

### CODE REVIEW

**Comment by Stanly5** [comment]
@duyplus 
Please check it again. It seems that something wrong in https://github.com/RaiDrive/RaiDrive.Translation/pull/313/commits/5a38a31def52b24d489f14dcec24f008db5c6020

File: StringDictionary.vi.xaml [hunk 1]
```diff
@@ -88,7 +88,8 @@
 	<system:String x:Key="Messages.Upgrade.NewVersion">Bạn có thể nâng cấp lên phiên bản {0}</system:String>
 	<system:String x:Key="Messages.Upgrade.Uptodate">Bạn đang sử dụng phiên bản mới nhất</system:String>
 	<system:String x:Key="Messages.VolumeInvalidChar">Tên không thể chứa những ký tự sau:</system:String>
-	<system:String x:Key="Messages.VolumeInvalidStr">Không sử dụng các tên sau:</system:String>
+	<system:String x:Key="Messages.VolumeInvalidStr">Không sử dụng các tên sau:
```
Comment (Stanly5) [inline StringDictionary.vi.xaml:91]: no change and missing </system:String>

File: StringDictionary.vi.xaml [hunk 2]
```diff
@@ -88,7 +88,8 @@
...
+	<system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần có mật khẩu để bao gồm thông tin xác thực.</system:String>
```
Comment (Stanly5) [inline StringDictionary.vi.xaml:92]: Incorrect line is added

Comment (duyplus) [reply StringDictionary.vi.xaml:92]: > Incorrect line is added

I have edited it again

Comment (Stanly5) [reply StringDictionary.vi.xaml:92]: Line#92 <system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần có mật khẩu để bao gồm thông tin xác thực.</system:String> is overlaps with line# 69 	<system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần phải có mật khẩu để bao gồm thông tin xác thực.</system:String>
It should be modified the content of line 69 without adding line 92.

File: StringDictionary.vi.xaml [hunk 3]
```diff
@@ -130,6 +131,9 @@
 	<system:String x:Key="Phrases.UpdateChangeAsync">Cập nhật các thay đổi về bộ nhớ không đồng bộ</system:String>
 	<system:String x:Key="Phrases.UseDriveIcon">Sử dụng biểu tượng tùy chỉnh trên một ổ đĩa được kết nối</system:String>
 	<system:String x:Key="Phrases.UnavailableOption">Tùy chọn không khả dụng</system:String>
+	<system:String x:Key="Phrases.ClearExistingConnection">Xóa bỏ kết nối hiện tại</system:String>
+	<system:String x:Key="Phrases.CredentialsIncluding">Bao gồm thông tin xác thực</system:String>
+	<system:String x:Key="Phrases.PasswordMandatory">Mật khẩu là bắt buộc</system:String>
```
[the remark this answers is not in the document]
Comment (Stanly5) [reply StringDictionary.vi.xaml:136]: Same as above Line#92, 
Line#134, 135, 136 are overlaps with line#95, 97, 119.

[the remark this answers is not in the document]
Comment (duyplus) [reply StringDictionary.vi.xaml:132]: I'm sorry, when I synced the fork, I forgot to check the new lines that had been added earlier. I have updated it once more.

**Pull request merged**